lib/converters/v2.0.0/converter-v2-to-v1.js

```diff
@@ -572,25 +539,22 @@ module.exports = {
             _.set(collection, 'info.id', util.uid());
         }
 
-        var units = ['order', 'folders_order', 'folders', 'requests'],
+        var tempCollection = _.cloneDeep(collection),
```

Comment:
@kunagpal I can see your intention to not mutate the input collection. Isn't this expensive?

Reply:
It is, but the transformer is not meant to mutate any incoming data, something that can be seen from the existing converters.

Reply:
ok